src/rockstor/storageadmin/static/storageadmin/js/views/snapshots.js

```diff
@@ -42,7 +42,7 @@ SnapshotsView = SnapshotsCommonView.extend({
 	this.snapshots = this.options.snapshots;
 	this.collection = new SnapshotsCollection();
 	this.shares = new ShareCollection();
-	this.collection.pageSize = RockStorGlobals.maxPageSize; 
+	this.collection.pageSize = RockStorGlobals.maxPageSize;
```

Comment:
Do we still need this? You've already set the pageSize to maxPageSize in the paginated collection(parent class), no?